### PR TITLE
title

### LINKED ISSUE #7: title
using props we will pass in the text 
here an example: 
https://github.com/acm-ucr/art-factory-website/blob/dev/src/components/about/About.jsx#L26
https://github.com/acm-ucr/art-factory-website/blob/dev/src/components/Title.jsx

Screenshot 2024-06-30 at 5 27 24 PM

### PR DESCRIPTION
**Pull request by manishgandi24**
Before:
Screenshot 2024-07-13 at 10 30 29 PM

After:
Screenshot 2024-07-15 at 7 19 34 PM

### CODE REVIEW

File: src/components/Title.jsx [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
 
+// const Title = ({ title }) => {
+//   return <div>{title}</div>;
+// };
+
+// export default Title;
 const Title = ({ title }) => {
-  return <div>Title</div>;
+  return (
+    <div className="flex-col inline-flex gap-3 items-center justify-center">
```
Comment (stormyy00) [inline src/components/Title.jsx]: you only need 1 div  you can remove the rest as well as make sure you use ais-blue-100 for the text color.

File: src/components/Title.jsx [hunk 2]
```diff
@@ -1,7 +1,21 @@
 import React from "react";
 
+// const Title = ({ title }) => {
```
Comment (stormyy00) [inline src/components/Title.jsx]: remove these comments

File: src/components/Title.jsx (hunk 1 above)
Comment (manishgandi24) [reply src/components/Title.jsx]: Which div do you want me to keep? Every time I remove a div, the code stops working.

Comment (stormyy00) [reply src/components/Title.jsx]: all the them 
you just to have you one div so like 
div (styling)> {title} div>

Comment (manishgandi24) [reply src/components/Title.jsx]: Screenshot 2024-07-16 at 12 40 10 PM
Is this what you want?

**Review by stormyy00 — APPROVED** [review]

**Pull request merged**